caddyhttp/gzip/gzip.go

```diff
@@ -161,7 +162,8 @@ func (w *gzipResponseWriter) CloseNotify() <-chan bool {
 	if cn, ok := w.ResponseWriter.(http.CloseNotifier); ok {
 		return cn.CloseNotify()
 	}
-	panic(httpserver.NonCloseNotifierError{Underlying: w.ResponseWriter})
+	log.Printf("[WARNING] gzipResponseWriter: %s\n", httpserver.NonCloseNotifierError{Underlying: w.ResponseWriter})
+	return nil
```

Comment:
So, this change is in a few places; do we know what caused it to not be a CloseNotifier, and why we were trying to CloseNotify something that couldn't be? (what caused the panic?)